osmaxx/excerptexport/models/export.py

```diff
@@ -1,21 +1,33 @@
 import logging
 
 from django.db import models
+from django.utils.datetime_safe import datetime
 from django.utils.translation import ugettext_lazy as _
 from rest_framework.reverse import reverse
 
 from osmaxx.conversion_api import statuses
 from osmaxx.conversion_api.formats import FORMAT_CHOICES
 from osmaxx.conversion_api.statuses import FAILED, FINAL_STATUSES, FINISHED
 
-INITIAL = 'initial'
-INITIAL_CHOICE = (INITIAL, _('initial'))
-STATUS_CHOICES = (INITIAL_CHOICE,) + statuses.STATUS_CHOICES
-
 logger = logging.getLogger(__name__)
 
 
-class Export(models.Model):
+class TimeStampModelMixin(models.Model):
+    created = models.DateTimeField(_('created on'), default=datetime.now, blank=True, editable=False)
+    updated = models.DateTimeField(_('updated on'), default=None, blank=True, editable=False, null=True)
```

Comment:
As these are datetimes, not bare dates, use "at" rather than "on" for the display name. You might want to rename the fields themselves to `created_at` and `updated_at`, too, as the current names seem to imply a boolean type.